### PR TITLE
Fix Firestore podspec overlay build

### PR DESCRIPTION
**Pull request by paulb777**
Add missing nanopb directory to podspec.  
Otherwise imports of headers in Protos/nanopb fail to compile.

### CODE REVIEW

File: FirebaseFirestore.podspec [hunk 1]
```diff
@@ -31,6 +31,7 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   s.source_files = [
     'Firestore/Source/**/*',
     'Firestore/Port/**/*',
+    'Firestore/Protos/nanopb/**/*.[hm]',
```
Comment (wilhuff) [inline FirebaseFirestore.podspec]: These are all .c files, not .m, so this should be `*.[hc]`.

**Comment by paulb777** [comment]
@wilhuff I had to add a C Flag to get the C files to build.  Should these files really be part of the library? They don't seem necessary for the unit tests.

Please reconfirm your LGTM.

**Comment by wilhuff** [comment]
So we could conceivably exclude these for now. To do so, we'd also need to exclude `Firestore/core/src/firebase/firestore/remote/serializer.{h,cc}`.

Background: we're in the midst of rewriting our serializer from using Objective-C protobuf to using nanopb. This work is incomplete though and the production code does not use this yet. There's no harm in including this code, and excluding it will make development slightly harder. However, development is currently proceeding mostly under the CMake build so we could exclude this without harming velocity.

Regarding this specific flag, it matches what we've done in the CMake build:

https://github.com/firebase/firebase-ios-sdk/blob/fe19fca0e521e3765e4ecf6a87b0a3d622a52b92/Firestore/Protos/CMakeLists.txt#L44

**Comment by paulb777** [comment]
Sounds good to optimize for best development. I found the flag in firebase-ios-sdk/Firestore/Protos/CMakeLists.txt :)

**Pull request merged**